samples/subsys/usb/console/arduino_101.overlay

```diff
@@ -1,10 +0,0 @@
-/*
- * Copyright (c) 2018 Intel Corporation
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-/ {
-        chosen {
-                zephyr,console = &usb_cdc;
```

Comment:
Do you need to add an app.overlay to set this?

Reply:
yep, forgot to 'git add' the file.